tests/output/c/02000-i2c-core.c

```diff
@@ -795,9 +795,9 @@ static int i2c_probe_address(struct i2c_adapter *adapter, int addr, int kind,
    return((err == -ENODEV) ? 0 : err);
 }
 
-int i2c_probe(struct i2c_adapter *adapter,
+int i2c_probe(struct i2c_adapter             *adapter,
               struct i2c_client_address_data *address_data,
-              int (*found_proc)(struct i2c_adapter *, int, int))
+              int (                          *found_proc)(struct i2c_adapter *, int, int))
```

Comment:
I don't think the alignment should be done in this case so no change should be the correct approach here.